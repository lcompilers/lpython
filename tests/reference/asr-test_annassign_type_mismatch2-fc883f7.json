{
    "basename": "asr-test_annassign_type_mismatch2-fc883f7",
    "cmd": "lpython --show-asr --no-color {infile} -o {outfile}",
    "infile": "tests/errors/test_annassign_type_mismatch2.py",
    "infile_hash": "21f89e1a00c786674c46a2a6bb76d9e9fce37e1d9e11532d5134f68d",
    "outfile": null,
    "outfile_hash": null,
    "stdout": null,
    "stdout_hash": null,
    "stderr": "asr-test_annassign_type_mismatch2-fc883f7.stderr",
    "stderr_hash": "873b4521e2155bc92405db41e7745a4fb5441b5d80f94467d7b1b637",
    "returncode": 2
}