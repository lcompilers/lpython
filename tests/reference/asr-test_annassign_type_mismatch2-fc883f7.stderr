semantic error: Type mismatch in annotation-assignment, the types must be compatible
 --> tests/errors/test_annassign_type_mismatch2.py:4:5
  |
4 |     x: f64[5] = [1.0, 2.0, 3.0, 4.0, 5.0]
  |     ^           ^^^^^^^^^^^^^^^^^^^^^^^^^ type mismatch ('f64[5]' and 'list[f64]')
